dependencies.yaml

```diff
@@ -562,7 +578,7 @@ dependencies:
     common:
       - output_types: conda
         packages:
-          - &ucx_conda_run ucx>=1.15.0
```

Comment:
I guess this anchor was just unused previously?

Reply:
yeah exactly. Maybe I should have left that out of the diff, but figured it was low-risk, non-controversial, and related-enough to include here.